### PR TITLE
Add blog post releasing spores

### PR DESCRIPTION
**Pull request by jvican**

### CODE REVIEW

**Comment by SethTisue** [comment]
It might be nice if the post weren't _quite_ so entirely Spark-centric. spores should be useful in other contexts as well. Perhaps tweak the intro to acknowledge that, and include something in the conclusion that invites other projects besides Spark with an interest in spores and serialization to participate?

File: blog/_posts/2016-11-30-spores-release.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+layout: blog
+post-type: blog
+by: Jorge Vicente Cantero
+title: "Releasing spores 0.4.1"
+---
+
+Today we’re announcing the release of spores and its transitive serializable
```
Comment (ashawley) [inline blog/_posts/2016-11-30-spores-release.md]: Is "we" the Scala Center?

**Comment by ashawley** [comment]
I wonder if Apache Spark would accept a blog post?

https://github.com/apache/spark-website

Or maybe they'd only allow a mention on this page:

http://spark.apache.org/third-party-projects.html

**Comment by heathermiller** [comment]
@jvican I've reviewed it and made some changes. LGTM! Thanks for writing this up!

@ashawley Good idea about posting on a Spark site. Though this is something that was an initiative of the Scala Center, so it's good that the Scala blog gets updated with a project that is close to the design of the language, especially since we're the ones that made it :)

**Pull request merged**